start/sw/web-app/laravel5.md

```diff
@@ -16,6 +16,7 @@ This guide will help you setup your local development environment for Laravel, a
 #### Install Nginx Web Server
 Laravel Valet and Laravel recommends using Nginx. Follow the nginx installation instructions [here](/start/sw/web-app/nginx.html).
 
+You can also choose [Apache](/start/sw/web-app/apache.html) as the web server. But it requires an additonal package, `php`, to serve PHP contents properly. After installing the package, Apache needs to be restarted to take effect.
```

Comment:
I still think the `php` related content is duplicated here, as the paragraph `#### Setup environment for PHP development` already comments on that. Also the `restart` sentence should probably be moved under the same PHP paragraph.